react/introduction/setting_up_a_react_environment.md

```diff
@@ -57,6 +57,14 @@ Fire up a terminal instance, `cd` over to the folder containing your projects, a
 npm create vite@latest my-first-react-app -- --template react
 ~~~
 
+If you see the following output, enter `y` and then press <kbd>enter</kbd>:
+
+~~~
+Need to install the following packages:
+  create-vite04.4.1
```

Comment:
I think we shouldn't be specific about the versioning otherwise we'd have to update the lesson for every minor/patch update to the `create-vite` package. For major updates, it's worth coming back and testing the command works as expected and then update the lesson.

```suggestion
  create-vite@4.X.X
```

Reply:
Good call! I should have thought about that instead of just pasting.